compiler-rt/test/asan/TestCases/Darwin/odr-lto.cpp

```diff
@@ -5,7 +5,7 @@
 
 // RUN: %clangxx_asan -DPART=0 -c %s -o %t-1.o -flto -mllvm -asan-use-private-alias
 // RUN: %clangxx_asan -DPART=1 -c %s -o %t-2.o -flto -mllvm -asan-use-private-alias
-// RUN: %clangxx_asan_lto %t-1.o %t-2.o -o %t -flto -mlinker-version=133
```

Comment:
Is this a case where the test needed a minimum version of the linker initially, but now that version is too old?

Reply:
This is explained in e770bc935b04c52e4e7ea7cd4a467d1fddfb4234.

"When building clang with lld, we don't get a default mlinker-version [1]. This causes us to not pass -lto_library to ld64 [2].
Explicitly pass -mlinker-version=133 so we properly pass -lto_library to ld64 and don't get LLVM bitcode version mismatches due to LTO."